eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/listeners/ClusteringProgressListener.java

```diff
@@ -41,16 +60,59 @@ public ExitStatus afterStep(StepExecution stepExecution) {
         String stepName = stepExecution.getStepName();
         long numTotalItemsRead = stepExecution.getReadCount();
         logger.info("Step {} finished: Items (ss) read = {}, rs created = {}, rs updated = {}, " +
-                            "rs merge operations = {}, ss kept unclustered = {}, " +
-                            "ss clustered = {}, ss updated rs merged = {}, ss update operations = {}",
-                    stepName, numTotalItemsRead,
-                    clusteringCounts.getClusteredVariantsCreated(),
-                    clusteringCounts.getClusteredVariantsUpdated(),
-                    clusteringCounts.getClusteredVariantsMergeOperationsWritten(),
-                    clusteringCounts.getSubmittedVariantsKeptUnclustered(),
-                    clusteringCounts.getSubmittedVariantsClustered(),
-                    clusteringCounts.getSubmittedVariantsUpdatedRs(),
-                    clusteringCounts.getSubmittedVariantsUpdateOperationWritten());
+                        "rs merge operations = {}, ss kept unclustered = {}, " +
+                        "ss clustered = {}, ss updated rs merged = {}, ss update operations = {}",
+                stepName, numTotalItemsRead,
+                clusteringCounts.getClusteredVariantsCreated(),
+                clusteringCounts.getClusteredVariantsUpdated(),
+                clusteringCounts.getClusteredVariantsMergeOperationsWritten(),
+                clusteringCounts.getSubmittedVariantsKeptUnclustered(),
+                clusteringCounts.getSubmittedVariantsClustered(),
+                clusteringCounts.getSubmittedVariantsUpdatedRs(),
+                clusteringCounts.getSubmittedVariantsUpdateOperationWritten());
+
+        try {
+            String assembly = stepExecution.getJobExecution().getJobParameters().getString("assemblyAccession");
```

Comment:
Could these be retrieved from an injected InputParameters bean?

Reply:
Updated.
Instead of passing just the chunksize, updated the class constructor to take Object InputParameters